inst/vpa.m

```diff
@@ -62,8 +62,15 @@
         'x, n = _ins'
         'return sympy.N(x, n),' };
     r = python_cmd (cmd, x, n);
-  elseif (ischar(x))
-    x = magic_str_str(x);
+  elseif (ischar (x))
+    if (strcmp (x, 'inf') || strcmp (x, 'Inf') || strcmp (x, '+inf') || ...
```

Comment:
Hi, i maybe instead of this vpa we can simplify to:
```
function r = vpa(x, n)

  if (nargin == 1)
    n = digits();
  elseif (nargin ~= 2)
    print_usage ();
  end

  n = sym(n);

  cmd = {
      'x, n = _ins'
      'return sympy.N(x, n),' };
  r = python_cmd (cmd, x, n);
end
```
Actually pass all the tests, and seems works.

All the check we do here are basically what sym function does.